tests/dummy/public/test-forms/dynamic-behaviour-v2/form.ttl

```diff
@@ -0,0 +1,143 @@
+# This form was generated using the form builder: https://poc-form-builder.relance.s.redpencil.io/formbuilder/6213B0B3B1D7CA00090000E2/edit
+
+@prefix form: <http://lblod.data.gift/vocabularies/forms/> .
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix fieldGroups: <http://data.lblod.info/field-groups/> .
+@prefix fields: <http://data.lblod.info/fields/> .
+@prefix displayTypes: <http://lblod.data.gift/display-types/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix lblodBesluit: <http://lblod.data.gift/vocabularies/besluit/>.
+@prefix schema: <http://schema.org/>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+
+##########################################################
+# form
+##########################################################
+
+form:6b70a6f0-cce2-4afe-81f5-5911f45b0b27 a form:Form, form:TopLevelForm ;
+    mu:uuid "6b70a6f0-cce2-4afe-81f5-5911f45b0b27" ;
+    form:includes fields:8e24d707-0e29-45b5-9bbf-a39e4fdb2c11 ;
+    form:includes fields:subset-property-group ;
+    # these feel redundant because they are form:partOf the sections
+    form:includes fields:fd40cdcf-d085-4b71-930f-0b1c05e021d9 ;
+    form:includes fields:43dd5003-c9a4-48c6-b650-b90f2bce8c7d ;
+    form:includes fields:6fda9f15-6f5c-4174-bb28-3fac0804cefa ;
+    form:includes fields:star-rating-subset ;
+    form:includes fields:more-comment-subset .
+
+##########################################################
+#  property-group
+##########################################################
+fields:8e24d707-0e29-45b5-9bbf-a39e4fdb2c11 a form:Section;
```

Comment:
If possible I think we would like to get rid of the uuid's as a name and use naming in words instead for example this one could be 
```
fields:8e24d707-0e29-45b5-9bbf-a39e4fdb2c11 a form:Section;

# to

fields:singleCodeListValueS a form:Section;
```
Adding the First letter of the type at the end of the name is done in the formBuilder (in this example `S` stand for `Section`) not sure if we want to make it a general way of naming?

Reply:
added a sections: prefix for extra clarity